Issues building

Here is my Dockerfile:
```
FROM ubuntu:20.04

# prep system
RUN ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime
RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y libpcre3-dev libssl-dev zlib1g-dev libprotobuf-dev protobuf-compiler perl make cmake build-essential curl unzip autoconf libtool pkg-config git

WORKDIR /tmp

# OpenTracing API for C++
RUN git clone -b v1.5.1 https://github.com/opentracing/opentracing-cpp.git && \
    mkdir opentracing-cpp/.build && \
    cd opentracing-cpp/.build && \
    cmake .. && \
    make && \
    make install && \
    cd ../.. && \
    rm -rf opentracing-cpp

# gRPC
RUN git clone -b v1.28.1 --recursive https://github.com/grpc/grpc && \
    mkdir -p grpc/cmake/build && \
    cd grpc/cmake/build && \
    cmake ../.. && \
    make && \
    make install && \
    cd ../../.. && \
    rm -rf grpc

# LightStep distributed tracing library for C++.
RUN git clone https://github.com/lightstep/lightstep-tracer-cpp.git && \
    mkdir lightstep-tracer-cpp/.build && \
    cd lightstep-tracer-cpp/.build && \
    cmake -DWITH_GRPC=ON -DWITH_DYNAMIC_LOAD=OFF .. && \
    make && \
    make install && \
    cd ../.. && \
    rm -rf lightstep-tracer-cpp
```

and here are the errors I get:
```
Cloning into 'lightstep-tracer-cpp'...
-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found Protobuf: /usr/local/bin/protoc-3.11.2.0 (found version "3.11.2.0") 
-- Found Protobuf: /usr/local/lib/libprotobuf.a (found version "3.11.2") 
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/lightstep-tracer-cpp/.build
Scanning dependencies of target lightstep_3rd_party
...
[100%] Linking CXX executable tutorial
/usr/bin/ld: ../../liblightstep_tracer.so.0: undefined reference to `lightstep::MetricsTracker::MetricsTracker(lightstep::MetricsObserver&)'
/usr/bin/ld: ../../liblightstep_tracer.so.0: undefined reference to `lightstep::MetricsTracker::OnFlush()'
/usr/bin/ld: ../../liblightstep_tracer.so.0: undefined reference to `lightstep::MetricsTracker::OnSpansSent(int)'
/usr/bin/ld: ../../liblightstep_tracer.so.0: undefined reference to `lightstep::MetricsTracker::ConsumeDroppedSpans()'
/usr/bin/ld: ../../liblightstep_tracer.so.0: undefined reference to `lightstep::MetricsTracker::UnconsumeDroppedSpans(int)'
collect2: error: ld returned 1 exit status
make[2]: *** [example/tutorial/CMakeFiles/tutorial.dir/build.make:109: example/tutorial/tutorial] Error 1
make[1]: *** [CMakeFiles/Makefile2:1135: example/tutorial/CMakeFiles/tutorial.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
```

Help appreciated.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -215,6 +215,7 @@ set(LIGHTSTEP_SRCS src/common/utility.cpp
                    src/recorder/fork_aware_recorder.cpp
                    src/recorder/legacy_manual_recorder.cpp
                    src/recorder/manual_recorder.cpp
+		   src/recorder/metrics_tracker.cpp
                    src/recorder/transporter.cpp
                    src/recorder/serialization/report_request.cpp
                    src/recorder/serialization/report_request_header.cpp

